tests/libtest/thrift_client_to_db.py

```diff
@@ -147,13 +149,15 @@ def _getAll_emu(self, func_name, *args):
 
 class CCAuthHelper(ThriftAPIHelper):
 
-    def __init__(self, host, port, uri, auto_handle_connection=True,
+    def __init__(self, version, host, port, uri, auto_handle_connection=True,
                  session_token=None):
         # Import only if necessary; some tests may not add this to PYTHONPATH.
         from libcodechecker import session_manager
-        from Authentication import codeCheckerAuthentication
 
-        transport = THttpClient.THttpClient(host, port, uri)
+        if version == '6.0':
```

Comment:
Is this a good idea to hardcode this constant? Is it a good idea to also rely on the minor version here?